{
  "comments": [
    {
      "key": {
        "uuid": "9a3d3df0_1cf848e4",
        "filename": "opendaylight/netconf/netconf-impl/src/test/java/org/opendaylight/netconf/impl/osgi/NetconfOperationRouterImplTest.java",
        "patchSetId": 1
      },
      "lineNbr": 49,
      "author": {
        "id": 84
      },
      "writtenOn": "2016-02-09T12:17:39Z",
      "side": 1,
      "message": "Putting this inside a static method annotated as @BeforeClass would save you the catch block. I think @BeforeClass method can throw any exception and JUnit will take care of it.",
      "range": {
        "startLine": 43,
        "startChar": 4,
        "endLine": 49,
        "endChar": 5
      },
      "revId": "3c410a07d38fcc1759562b1832d940dddfca24e9",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "9a3d3df0_bce83c11",
        "filename": "opendaylight/netconf/netconf-impl/src/test/java/org/opendaylight/netconf/impl/osgi/NetconfOperationRouterImplTest.java",
        "patchSetId": 1
      },
      "lineNbr": 104,
      "author": {
        "id": 84
      },
      "writtenOn": "2016-02-09T12:17:39Z",
      "side": 1,
      "message": "Asserting exception messages tends to be fragile. DocumentedException provides plenty of additional information e.g. error-type, error-severity ... why not assert those instead of message ?",
      "range": {
        "startLine": 103,
        "startChar": 0,
        "endLine": 104,
        "endChar": 51
      },
      "revId": "3c410a07d38fcc1759562b1832d940dddfca24e9",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81",
      "unresolved": false
    }
  ]
}